### ISSUE TITLE
Cannot collapse by domain the content in the fourth section of the matrix (blacklisted hostnames)

### ISSUE THREAD

**Issue by darkred** [gorhill/uMatrix#952]
_Using uMatrix 1.3.2/1.3.3b8 in Chrome 64 and Firefox 58 (win10x64)._

**The issue is that currently it's not possible to collapse by base domain the fourth section of the matrix, i.e. the blacklisted hostnames.**

e.g. here is the matrix for the current URL:
![2018-02-13_015104](https://user-images.githubusercontent.com/723651/36126302-8d88869a-1060-11e8-9f32-509e6c4be308.jpg)

and here it is as you expand the blacklisted content section:
![2018-02-13_015008](https://user-images.githubusercontent.com/723651/36126325-a3b5d008-1060-11e8-8b1e-cd19a70bec72.jpg)

i.e. the line `2 blacklisted hostname(s)` , expanded becomes 4 entries, instead of 2 collapsed ones: `*.githubapp.com` and `*.google-analytics.com`.
In other words, there are no "collapse" buttons for this section of the matrix when you expand it.

<br/>

PS. Also, the 3 'avatars_..githubusercontent.com' entries, 
they are not displayed grouped/collapsed as a single one: `*.githubusercontent.com `. 
Shouldn't they be displayed so, i.e. like in the entry above them, `*.github.com` ?

**Comment by gorhill** [comment]
By design.

**Comment by gorhill** [comment]
> Also, the 3 'avatars_..githubusercontent.com' entries

`githubusercontent.com` is a public suffix.

**Comment by darkred** [comment]
@gorhill Thanks for replying.

_And regarding the PS,  thanks for the info and sorry for not managing to find the existing issues #851, #155 and https://github.com/gorhill/uMatrix/issues/54#issuecomment-62480877_.
